test_runner/regress/test_lfc_resize.py

```diff
@@ -103,3 +105,21 @@ def get_lfc_size() -> tuple[int, int]:
         time.sleep(1)
 
     assert int(lfc_file_blocks) <= 128 * 1024
+
+    nretries = 10
```

Comment:
please add a brief comment to explain what this new code tests

Reply:
Done